docs/source/tutorial.rst

```diff
@@ -857,6 +863,9 @@ will send some data at our echo server and get responses back:
 .. literalinclude:: tutorial/echo-client.py
    :linenos:
 
+Note that this code will not work without a TCP server listening at 
+``127.0.0.1:12345``.
```

Comment:
I don't think the exact port number matters much, and it's useful to mention that we'll write a server below, so here's my suggestion:

```suggestion
Note that this code will not work without a TCP server such as the one
we'll implement below.
```